src/components/base/index.js

```diff
@@ -55,6 +60,14 @@ var Base = ffwdme.Class.extend({
 
     grid.x && el.addClass('ffwdme-grid-x' + grid.x);
     grid.y && el.addClass('ffwdme-grid-y' + grid.y);
+    if (grid.w) {
```

Comment:
I don't understand the purpose of this. 
The CSS class is defined in the class definition, but also gets overwritten by the values in the initializer? (but with the same values at the moment)

Reply:
the class gets appended I think so it was throwing a formatting error when I had a w-5 and a w-8 because  it couldn't decide what was happening with width.